.pre-commit-hooks.yaml

```diff
@@ -10,6 +10,12 @@
     language: pygrep
     entry: 'assert .*\.(not_called|called_)'
     types: [python]
+-   id: python-noeval
+    name: Check for eval()
```

Comment:
for consistency, the rest of the hook names are lower cased

Reply:
Actually, they aren't: https://github.com/pre-commit/pygrep-hooks/blob/master/.pre-commit-hooks.yaml#L2

But if you like I can modify that line as well. But I'm going to have to do that later this afternoon.

Reply:
yeah let's fix that one too, it was inconsistent and I just didn't notice it on PR :(